plaid/sandbox/client.py

```diff
@@ -33,12 +41,43 @@ def _load_fixture(filename):
         with open(os.path.join(dir_name, 'fixtures', filename)) as f:
             return json.loads(f.read())
 
+    @staticmethod
+    def _post_webhook(account_number, event_type):
+        account = SandboxClient.accounts[account_number]
+        data = SandboxClient._load_fixture('webhook/%s.json' % event_type)
+        data['access_token'] = 'test_%d' % account_number
+        headers = {'Content-type': 'application/json', 'Accept': 'application/json'}
+        requests.post(account['webhook'], data=json.dumps(data), headers=headers)
+
+    def _process_connect_success(self, data, account_number=None):
+        if not account_number:
+            account_number = int(self.access_token[5:])
+        account = SandboxClient.accounts[account_number]
+
+        if account['login_only']:
+            del data['transactions']
+        if account['webhook'] is not None:
+            gevent.spawn_later(1, self._post_webhook, account_number, 'initial')
+            gevent.spawn_later(6, self._post_webhook, account_number, 'historical')
```

Comment:
Instead of making this function spawn greenlets we should really just expose a settable callback function for processing connect success.  On real sandbox environments we could pass in a callback that delays the request.post using celery. For local testing/behave we could use gevent. The point is that the user of the SandboxClient can decide what they want to to with the connect success data.  It also removes the introduction on an additional dependency.

Ultimately I think this will allow us more control over these delayed actions and since we can provide our own callback we can use our normal logging services to make sure we know when the post is failing for unexpected reasons.